doc/rados/operations/change-mon-elections.rst

```diff
@@ -1,88 +1,100 @@
 .. _changing_monitor_elections:
 
-=====================================
-Configure Monitor Election Strategies
-=====================================
+=======================================
+Configuring Monitor Election Strategies
+=======================================
 
-By default, the monitors will use the ``classic`` mode. We
-recommend that you stay in this mode unless you have a very specific reason.
+By default, the monitors are in ``classic`` mode. We recommend staying in this
+mode unless you have a very specific reason.
 
-If you want to switch modes BEFORE constructing the cluster, change
-the ``mon election default strategy`` option. This option is an integer value:
+If you want to switch modes BEFORE constructing the cluster, change the ``mon
+election default strategy`` option. This option takes an integer value:
 
-* 1 for "classic"
-* 2 for "disallow"
-* 3 for "connectivity"
+* ``1`` for ``classic``
+* ``2`` for ``disallow``
+* ``3`` for ``connectivity``
 
-Once your cluster is running, you can change strategies by running ::
+After your cluster has started running, you can change strategies by running a
+command of the following form:
 
   $ ceph mon set election_strategy {classic|disallow|connectivity}
 
 Choosing a mode
 ===============
-The modes other than classic provide different features. We recommend
-you stay in classic mode if you don't need the extra features as it is
-the simplest mode.
 
-The disallow Mode
-=================
-This mode lets you mark monitors as disallowed, in which case they will
-participate in the quorum and serve clients, but cannot be elected leader. You
-may wish to use this if you have some monitors which are known to be far away
-from clients.
-You can disallow a leader by running:
+The modes other than ``classic`` provide extra features. We recommend staying
+in ``classic`` mode if you don't need these extra features because it is the
+simplest mode.
```

Comment:
Suggest s/extra/specific/ so the uncareful reader is perhaps less inclined to disregard advice.

Reply:
I think all three of these suggestions are clear improvements. They're all accepted.